Sources/Widgets/Widgets3D/PaintWidget/example/index.js

```diff
@@ -436,11 +446,15 @@ scene.circleHandle.onInteractionEvent(() => {
     .getWidgetState()
     .getEllipseHandle()
     .getOrigin();
-  const scale3 = scene.circleHandle
+  const point2 = scene.circleHandle
     .getWidgetState()
-    .getEllipseHandle()
-    .getScale3();
-  painter.paintEllipse(center, scale3);
+    .getPoint1Handle()
+    .getOrigin();
+
+  const radius = vec3.distance(center, point2);
+
+  const corner = [radius, radius, radius];
+  painter.paintEllipse(center, corner);
```

Comment:
Why?

Reply:
Here i've tested solutions and found that this one is working. The painting tool was broken due to the change of scale3 value. Since i have a vague comprehension of the impact of this change, i used the computations of the computeScale3 in the painting tool.

Reply:
I don't understand exactly so I'll test and report back. Thanks for the explanation.

Reply:
It is not very clear to me, apologies for the shaky explanations. I recommend testing scale3 mixin value before and after the orientation update

Reply:
FYI: the changes I did here when switching to orientation did not work (my bad), so reverting as you're doing here makes sense for now, though it won't support orientation, so I created an issue to follow-up https://github.com/Kitware/vtk-js/issues/1734.

Resolving this thread in the meantime. Thanks